content/en/security_platform/guide/how-appsec-works.md

```diff
@@ -0,0 +1,68 @@
+
+### Compatibility
+
+For Datadog Application Security to be compatible with your Datadog configuration, you must have APM enabled, and [send traces to Datadog][1]. It uses the same libraries used by the APM suite, so you don't need to deploy and maintain another library for this task. Steps to enable Datadog Application Security are specific to runtime language. Check to see if your language is supported in the [Application Security prerequisites][2].
+
+### Performance
+
+Datadog Application Security uses processes already contained in the Agent and APM, so there are no performance implications when using it. Datadog Application Security collects and enriches trace data from APM. It doesn't require extra overhead for processes like analyzing attack attempts, because it matches against known patterns and flags them. All processing is done platform-side, after traces have been flagged. 
+
+
+### Data privacy
+
+There are multiple methods used to avoid your sensitive information being indexed. To take further action, you can set up [custom and static scrubbers][3], and use [exclusion filters][4].
+
+The [Sensitive Data Scanner][5] automatically scans traces that have been flagged by Datadog Application Security.
+
+**Note:** Datadog Application Security does not automatically obfuscate sensitive information or PII. To keep this sensitive data from being sent to Datadog, [configure the Datadog Agent or Tracer for data security][3].
+
+Contact Support to delete sensitive data that may have been indexed.
+
+## Threat detection methods
+
+Datadog uses multiple pattern sources, including the [OWASP ModSecurity Core Rule Set][6] to detect known threats and vulnerabilities in HTTP requests. When an HTTP request matches one of [the OOTB detection rules][7], a security signal is generated in Datadog.
+
+Security Signals are automatically created when Datadog detects meaningful attacks targeting your production services. It provides you with visibility on the attackers and the targeted services. You can set custom detection rules with thresholds to determine which attacks you want to be notified about.
+
+## Coverage
+
+Datadog Application Security categorizes attack attempts into different threat types:
+
+* **Unqualified attacks** match inbound HTTP requests with known attack patterns, but don't have any correlation to the business-logic.
+* **Contextualized attacks** have a correlation between known attack patterns and business-logic.
+* A **Vulnerability is triggered** when there is a correlation between attack patterns, and there's evidence of a successful vulnerability trigger.
+
+Datadog Application Security includes over 100 attack patterns that help protect against [many different kinds of attacks][8], including the following:
```

Comment:
Datadog Application Security includes over 100 attack patterns that help protect against [many different kinds of attacks][8], covering the following _vulnerabilities_: